src/osemgrep/cli_install_semgrep_pro/Install_semgrep_pro_subcommand.ml

```diff
@@ -75,24 +90,17 @@ let download_semgrep_pro (caps : < Cap.network ; .. >) platform_kind dest =
               m
                 "Visit https://semgrep.dev/products/pro-engine for more \
                  information.");
-          false
+          Lwt.return_false
       (* THINK: ??? is this raise for status? *)
```

Comment:
Still relevant?

Reply:
not sure, see other replies in this file w.r.t. reorganizing